docs/user/algo_rl2.md

```diff
@@ -0,0 +1,52 @@
+<!-- markdownlint-disable no-inline-html -->
+# RL<sup>2</sup>
+
+```eval_rst
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+   :widths: auto
+
+   * - **Paper**
+     - RL\ :sup:`2` : Fast Reinforcement Learning via Slow Reinforcement
+       Learning :cite:`duan2016rl`
+   * - **Framework(s)**
+     - .. figure:: ./images/tensorflow.png
+        :scale: 20%
+
+        TensorFlow
+   * - **API Reference**
+     - `garage.tf.algos.RL2 <../_autoapi/garage/tf/algos/index.html#garage.tf.algos.RL2>`_
+   * - **Code**
+     - `garage/tf/algos/rl2.py <https://github.com/rlworkgroup/garage/blob/master/src/garage/tf/algos/rl2.py>`_
+   * - **Examples**
+     - `examples <algo_rl2.html#examples>`_
+```
+
+When sampling for RL<sup>2</sup>, there are more than one environments to be
+sampled from. In the original implementation, within each trial, all episodes
+sampled will be concatenated into one single episode, and fed to the
+inner algorithm. Thus, returns and advantages are calculated across the
+episode.
+
+User should not instantiate RL2 directly. Currently garage supports [PPO](https://github.com/rlworkgroup/garage/blob/master/src/garage/tf/algos/rl2ppo.py)
```

Comment:
this is confusing on its own -- maybe you should add a subsection on RL2PPO and RL2TRPO